gdk4/Gir.toml

```diff
@@ -252,7 +251,7 @@ generate_display_trait = false
     manual = true # remove once https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3488 gets merged
     [[object.function]]
     name = "print"
-    manual = true #mutability issue
+    ignore = true #mutability issue
```

Comment:
Please update the comment to the reason why it's ignored. `# Same as ContentFormats::to_string` or something like that